net/adoptopenjdk/bumblebench/string/StringValueOfBench.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+package net.adoptopenjdk.bumblebench.string;
+
+import net.adoptopenjdk.bumblebench.core.MicroBench;
+
+import java.util.Random;
+
+/**
+ *
+ * String.valueOf can be accelerated by vector instructions.
+ *
+ * This benchmark generates a char array of fixed lengths and invokes
+ * valueOf to measure performance.
+ *
+ * The reported score is in terms of number of string valueOfs per sec.
+ *
+ * */
+
+public final class StringValueOfBench extends MicroBench {
```

Comment:
Have you explore JMH? It feels like this type of bench would better fit in that framework

Reply:
For karianna's awareness: https://github.com/eclipse-openj9/openj9/issues/4649#issuecomment-461889329

Reply:
Ah, OK, yeah OpenJ9 is different!